examples/plugins/cmd-runner/README.md

```diff
@@ -0,0 +1,27 @@
+# Command Runner
+
+This example creates an Sonobuoy plugin which runs each of its arguments (via /bin/sh) and saves the output in a results file.
+
+Kubectl is added to the image so that it can grab information from/about the cluster.
+
+All of the results files are added into a tar file and then the 'done' file is written, telling the Sonobuoy worker where to find the output.
+
+## Example
+
+```
+$ docker build . -t user/easy-sonobuoy-cmds:v0.1
+$ docker push user/easy-sonobuoy-cmds:v0.1
+
+$ sonobuoy gen plugin \
+--name=hello-world \
+--image user/easy-sonobuoy-cmds:v0.1 \
+--arg="echo hello world" \
+--arg="kubectl cluster-info" > hello-world.yaml
+
+$ sonobuoy run --plugin hello-world.yaml
+
+$ outfile=$(sonobuoy retrieve) && \
+mkdir results && tar -xf $outfile -C results &&
+cat results/plugins/hello-world/results/out*
```

Comment:
Maybe show where the output would be? Or describe where it is in the result output?

Reply:
Split up and clarified this a section a bit so they separate out getting the results vs printing these results specifically.

Also clarified the file naming for output at the top of this file.